stories/ChatSuggestions.stories.tsx

```diff
@@ -0,0 +1,541 @@
+      response:
+        'Machine learning is a subset of artificial intelligence that enables systems to learn and improve from experience.',
+      nextSuggestions: [
+        { id: '4', content: 'What are the types of machine learning?' },
+        { id: '5', content: 'How is ML different from AI?' },
+        { id: '6', content: 'What are common ML algorithms?' }
+      ]
+    },
+    'Explain neural networks': {
+      response:
+        'Neural networks are computing systems inspired by biological neural networks in the human brain.',
+      nextSuggestions: [
+        { id: '7', content: 'How do neurons work in a network?' },
+        { id: '8', content: 'What is backpropagation?' },
+        { id: '9', content: 'What are activation functions?' }
+      ]
+    },
+    'What is deep learning?': {
+      response:
+        'Deep learning is a type of machine learning based on artificial neural networks with multiple layers.',
+      nextSuggestions: [
+        { id: '10', content: 'What are CNNs?' },
+        { id: '11', content: 'What are RNNs?' },
+        { id: '12', content: 'What is a transformer model?' }
+      ]
+    }
+  };
+
+  const handleSendMessage = (message: string) => {
+    const mapping = suggestionResponseMap[message];
+    const response =
+      mapping?.response ||
+      'Thank you for your question. Here is a helpful response.';
+    const nextSuggestions = mapping?.nextSuggestions || [
+      { id: Date.now().toString(), content: 'Tell me more' },
+      { id: (Date.now() + 1).toString(), content: 'Can you clarify?' }
+    ];
+
+    setSessions(prev =>
+      prev.map(session =>
+        session.id === activeId
+          ? {
+              ...session,
+              conversations: [
+                ...session.conversations,
+                {
+                  id: Date.now().toString(),
+                  question: message,
+                  response,
+                  createdAt: new Date()
+                }
+              ]
+            }
+          : session
+      )
+    );
+
+    setSuggestions(nextSuggestions);
+  };
+
+  return (
+    <div
+      className="dark:bg-gray-950 bg-white"
+      style={{
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0,
+        padding: 20,
+        margin: 20,
+        borderRadius: 5
+      }}
+    >
+      <Chat
+        viewType="console"
+        sessions={sessions}
+        activeSessionId={activeId}
+        onSelectSession={setActiveId}
+        onSendMessage={handleSendMessage}
+        onDeleteSession={() => alert('delete!')}
+      >
+        <SessionsList>
+          <NewSessionButton />
+          <SessionGroups />
+        </SessionsList>
+        <SessionMessagePanel>
+          <SessionMessagesHeader />
+          <SessionMessages />
+          <ChatSuggestions
+            suggestions={suggestions}
+            onSuggestionClick={suggestion =>
+              console.log('Suggestion clicked:', suggestion)
```

Comment:
Would be great to set selected suggestion to the `ChatInput` because I don't see a proctical reason of displaying this suggestions without ability to select it.